flow/layers/image_filter_layer.cc

```diff
@@ -85,6 +86,7 @@ void ImageFilterLayer::Preroll(PrerollContext* context,
       // switch during the Paint phase even if they are not stable.
       // This benefit is seen most during animations.
       TryToPrepareRasterCache(context, GetCacheableChild(), matrix);
+      context->save_layer_count += 1;
```

Comment:
I just noticed this. Even though I am pushing for switching back to tagging the saveLayer calls themselves with events as it is more accurate and less disruptive, I thought I'd point out a problem with this specific change if for no other reason than to educate.

This call is preparing the children for a cache entry. There is not implicit saveLayer involved when those children are rasterized like there is when "this" is cached. So, this specific increment is incorrect in all cases.